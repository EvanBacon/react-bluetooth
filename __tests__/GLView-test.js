import React from 'react';
import renderer from 'react-test-renderer';

import GLView from '../lib/GLView';

describe('GLView', () => {
  describe('Platform.default', () => {
    it('renders', () => {
      const tree = renderer.create(<GLView />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
