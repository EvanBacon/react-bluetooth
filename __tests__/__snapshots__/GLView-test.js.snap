// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GLView Platform.default renders 1`] = `
<View
  onContextCreate={[Function]}
/>
`;
